src/modelbuilder_base.nit

```diff
@@ -213,6 +213,141 @@ class ModelBuilder
 		end
 		return res
 	end
+
+	# Return the static type associated to the node `ntype`.
+	# `mmodule` and `mclassdef` is the context where the call is made (used to understand formal types)
```

Comment:
Also, you commented each params except `with_virtual` which seems to affect the behavior.